### ISSUE TITLE
Did you managed to fix Black Screen Issue?

### ISSUE THREAD

**Issue by kasem-sm** [AndroidDeveloperLB/OpenGlVideoLiveWallpaper#1]
Hey, Whenever I search up a question of Live Wallpaper or any related issue and read reddit/stackoverflow posts, I've always seen you around & btw your fork of Alynx is helpful as well. 

The issue is some device shows a complete black screen when we apply the wallpaper, in the preview screen as well. It's not that the wallpaper goes black suddenly but it doesn't renders. did you managed to fix that? Thank You!

**Comment by AndroidDeveloperLB** [comment]
Sadly I lack the knowledge and experience with OpenGL in general, let alone in a live wallpaper.
That's why I tried to at least check other samples.
There isn't much information, tutorials and samples about OpenGL on live wallpaper apps (and live wallpapers in general too).

About Alynx, I tried to convert it to a simpler code so that I would understand it better.
My goal was:

- A minimal live wallpaper app, without shiny frameworks
- Loads a video to be shown
- Can switch to a new video/image file without any step needed from the user.
- Center-crop the content
- Allows to scroll 

I also tried to ask ChatGpt and Bard about this, but sadly they are terrible to work with at the moment for such a task. 
I even tried to use LibGdx to do it, but it's even harder for me to do the research of how to use video there.

If you can make it work with the above goal that I've written, please post a new repository. I can donate too if you wish.

**Comment by kasem-sm** [comment]
If I am not mistaken, you have an app on Google Play Store which supports
center crop and other scaling methods, right?

**Comment by AndroidDeveloperLB** [comment]
Yes, but it uses Canvas which is not efficient for complex stuff.
I've noticed it when I tried to add animation support to it.

OpenGL works differently. 
Many years ago I've studied OpenGL 1.x, but it changed a lot on 2.0 and now there is even 3.x and more...
So my old knowledge is probably worthless. Plus I forgot a lot of it.
